wherehows-web/app/components/dataset-compliance-field-tag.ts

```diff
@@ -164,7 +164,10 @@ export default class DatasetComplianceFieldTag extends Component {
    * @type {ComputedProperty<boolean>}
    */
   showCustomInput = computed('tag.logicalType', function(this: DatasetComplianceFieldTag): boolean {
-    const { logicalType } = get(this, 'tag');
+    const { logicalType, valuePattern = '' } = get(this, 'tag');
+
+    this.actions.tagValuePatternDidChange.call(this, valuePattern);
```

Comment:
It is changing, if the logicalType is set, the valuePattern will initially be undefined, this sets it to an empty string, and call the "onChange" action with the new value. If there is already a value, then it will essentially noop